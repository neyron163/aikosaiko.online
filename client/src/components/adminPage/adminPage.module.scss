@import '../styleguide/variables.scss';

:local {
    .container {
        display: flex;
        background-color: var(--color-alabaster);
        min-height: 100vh;
    }
}
