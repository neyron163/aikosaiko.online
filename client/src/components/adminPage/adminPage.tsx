import React from 'react';

import {SideBar} from './sideBar/sideBar';
import {ProductsQuery} from './products/products';
import {BrowserRouter as Router} from 'react-router-dom';

import s from './adminPage.module.scss';

/**
 * AdminPage
 */
export const AdminPage = () => (
    <div className={s.container}>
        <Router>
            <SideBar />
            <ProductsQuery />
        </Router>
    </div>
);
