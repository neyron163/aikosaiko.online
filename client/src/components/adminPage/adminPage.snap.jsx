import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router';

import {AdminPage} from './adminPage';

jest.mock('./sideBar/sideBar', () => ({SideBar: 'SideBar'}));
jest.mock('./products/products', () => ({ProductsQuery: 'ProductsQuery'}));

describe('<AdminPage />', () => {
    it('should render correctly', () => {
        const tree = renderer.create(
            <MemoryRouter initialEntries={['/']}>
                <AdminPage />
            </MemoryRouter>,
        );
        expect(tree.toJSON()).toMatchSnapshot();
    });
});
