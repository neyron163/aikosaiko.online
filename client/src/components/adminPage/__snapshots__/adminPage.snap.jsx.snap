// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminPage /> should render correctly 1`] = `
<div
  className="container"
>
  <SideBar />
  <ProductsQuery />
</div>
`;
